prog/mutation.go

```diff
@@ -426,18 +502,41 @@ func (ma *mutationArgs) collectArg(arg Arg, ctx *ArgCtx) {
 		if typ.Kind == BufferString && len(typ.Values) == 1 {
 			return // string const
 		}
+		if typ.Kind == BufferFilename || typ.Kind == BufferText {
+			priority = 60
```

Comment:
I find it somewhat hard to compare and reason about these numbers like 50, 60, 55, etc. Also it seems that the difference between, say, 50 and 55 is pretty minimal. I can understand if something needs to be mutated twice as frequently as something else. But for 50 and 55 the difference seems to be almost unnoticeable, why one should be mutated just a tiny bit more frequently than another?
What I would do is introduce a "unit priority", that probably should be mutation of a bool, let's say it's equal to 1.0. Then if something is 5.0 then the physical meaning is "mutated 5 times more often than a bool". Which looks like a useful way to reason about things to me. I would expect most priorities either be several times larger than one another, or be equal if we don't have reasons to think that one needs to be mutated significantly more often. Since things are very vague, I have hard time justifying that something needs to be mutated just 5% more often than something else.